### PR TITLE
create custom image input field for new campaign creation. (#18)

### PR DESCRIPTION
**Pull request by ihab4real**
issues: #18 

image

### CODE REVIEW

File: lib/pages/campaign_form_page.dart [hunk 1]
```diff
@@ -101,18 +104,24 @@ class _CampaignFormPageState extends State<CampaignFormPage> {
                 const SizedBox(
                   height: 20,
                 ),
+                const SizedBox(
+                  height: 20,
+                ),
+                const CustomImagePicker(),
```
Comment (Yassa-hue) [inline lib/pages/campaign_form_page.dart:110]: Add `onImageIsSelected` method which return the image file:
```
CustomImagePicker(
  onImageIsSelected: (imageFile) => {
    setState(() => _imageFile = imageFile)
  }
)
```

**Review by Yassa-hue — CHANGES_REQUESTED** [review]
Minor change.

File: lib/pages/campaign_form_page.dart (hunk 1 above)
Comment (ihab4real) [reply lib/pages/campaign_form_page.dart]: check it out

**Review by Yassa-hue — APPROVED** [review]
LGTM

**Pull request merged**